templates/resources/vmaas_load_balancer.md.tmpl

```diff
@@ -0,0 +1,25 @@
+---
+layout: ""
+page_title: "hpegl_vmaas_load_balancer Resource - vmaas-terraform-resources"
+subcategory: {{ $arr := split .Name "_" }}"{{ index $arr 1 }}"
+description: |-
+  {{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# Resource hpegl_vmaas_load_balancer
+
+-> Compatible version >= 5.4.6
+
+{{ .Description | trimspace }}
+`hpegl_vmaas_load_balancer` resource supports NSX-T Load Balancer creation.
+
+-> It is mandatory to choose the `Tier1 Gateway` while creating the load balancer
+(for this purpose `hpegl_vmaas_router` can be used).
+
+-> Tier1 Gateway Data Source `hpegl_vmaas_router` is supported for NSX-T Load Balancer.
```

Comment:
Tier1 Gateway Data Source `hpegl_vmaas_router` is supported for NSX-T Load Balancer is not needed.

Reply:
removed